File-Formats/PDF/Working-with-Security.md

```diff
@@ -1336,3 +1336,89 @@ Returns user password</td><td>
 Returns owner Password (owner password is same as the user password; it allows full permission to users).</td></tr>
 </tbody>
 </table>
+
+## Troubleshooting
+
+ 1. If the document is protected with the owner's password only, it can be opened without entering a password. In that case, can the Syncfusion library remove or change restrictions? 
+ 
+      Using our Syncfusion library, remove or edit permissions in a document restricted by an owner password using our [import pages option](https://help.syncfusion.com/file-formats/pdf/working-with-pages#importing-pages-from-an-existing-document).
+ 
+ 2. How do you open a document using the owner's password if it's different from the user's password?
+
+      Open a document using the owner's password if it's different from the user's password. You can  refer to the following code example.
```

Comment:
You can refer to the following code to open a document using owner passed and edit the security permissions.